src/utils.rs

```diff
@@ -1,33 +1,63 @@
 use crate::AppSW;
 use core::str::from_utf8;
 
-pub const MAX_ALLOWED_PATH_LEN: usize = 10;
```

Comment:
Why not keep this (maybe renamed to `DEFAULT_MAX_PATH_LEN`) instead of having the `10` as a literal value defined on line 9 ?

Reply:
There is now only one single use of this value, and this constant is not supposed to be used elsewhere anymore. Having a constant won't help to factorize the code, and code is easier to read by removing it. I don't see a good reason to keep it.

Also it is not really supposed to be modified: since it is a generic argument of `Bip32Path`, if a different value is to be used it can be done explicitely like `Bip32Path<12>`.

If you still want to have a constant I can make the change, but I will make it non pub.

Reply:
Good points. Let's keep the literal `10` then.